src/ID/include/BiomechanicalAnalysis/ID/InverseDynamics.h

```diff
@@ -18,6 +18,8 @@
 #include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
 #include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
 
+#include <ResolveRoboticsURICpp.h>
```

Comment:
I would move this in the .cpp and move `ResolveRoboticsURICpp` to the `PRIVATE_LINK_LIBRARIES`, so you can install the library without installing `ResolveRoboticsURICpp` .

Reply:
Thank you, I did it in [a14fd1e](https://github.com/ami-iit/biomechanical-analysis-framework/pull/35/commits/a14fd1ed9aa44bd92561230fed13dca4020d88b9)